project.clj

```diff
@@ -1,46 +1,50 @@
+(def clj-version "1.10.1")
+(def metrics-clojure-version "2.10.0")
+
 (defproject threatgrid/clj-momo "0.3.4-SNAPSHOT"
   :description "Library code produced by the Cisco ThreatGrid team for building swagger backed API services"
   :url "https://github.com/threatgrid/clj-momo"
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
-  :dependencies [;; logging
+  :pedantic? :abort
+  ; when changing, use `lein pom; mvn dependency:tree -Dverbose` to print
+  ; full deps tree and understand all conflicts.
+  ; Also see `https://github.com/clj-commons/vizdeps` for visual representation
+  ; (may require a local install, not on clojars).
+  ; Use :exclusions only to permanently delete a dependency, and document reasons.
+  ; Avoid combining :exclusions with an override as it can obscure the indended
+  ; dependencies when browsing the full deps tree.
+  :dependencies [[org.clojure/clojure ~clj-version]
                  [org.clojure/tools.logging "0.5.0"]
-
-                 ;; schemas
                  [prismatic/schema "1.1.12"]
                  [metosin/schema-tools "0.12.2"]
-
-                 ;; time
                  [com.andrewmcveigh/cljs-time "0.5.2"]
                  [clj-time "0.15.2"]
-
-                 ;; uri
                  [com.arohner/uri "0.1.2"]
-
-                 ;;json
                  [cheshire "5.9.0"]
-
-                 ;;http
-                 [clj-http "3.10.0"]
-
-                 ;; Metrics
-                 [metrics-clojure "2.10.0"]
-                 [metrics-clojure-jvm "2.10.0"]
-                 [metrics-clojure-ring "2.10.0"]
-                 [metrics-clojure-riemann "2.10.0"]
-                 [clout "2.2.1"]
+                 [clj-http "3.10.0"] ;TODO bump clj-http with https://github.com/dakrone/clj-http/pull/532
                  [slugger "1.0.1"]
-                 [riemann-clojure-client "0.5.1"
-                  ;; Protobuf-java is brought in by ClojureScript
-                  :exclusions [com.google.protobuf/protobuf-java]]]
+                 [riemann-clojure-client "0.5.1"]
+                 [metrics-clojure ~metrics-clojure-version]
+                 [metrics-clojure-jvm ~metrics-clojure-version]
+                 [metrics-clojure-ring ~metrics-clojure-version]
+                 [metrics-clojure-riemann ~metrics-clojure-version
+                  :exclusions [;renamed to (and incompatible with) `io.riemann/riemann-java-client`,
+                               ;which is provided by `riemann-clojure-client`.
+                               com.aphyr/riemann-java-client]]
```

Comment:
Is it possible to use `[threatgrid/metrics-clojure-riemann "2.10.1"]` like you did in the IROH repo?

Reply:
Ugh, yes great point.

Reply:
Updated to `[threatgrid/metrics-clojure-riemann "2.10.1"]`.